ols/app/models/models.py

```diff
@@ -502,6 +502,7 @@ class CacheEntry(BaseModel):
 
     query: str
     response: Optional[str] = ""
+    attachments: list[Attachment] = Field(default_factory=list)
```

Comment:
I don't try it, but `= []` is not sufficient?

Reply:
It should work too, pydantic is smart in this way. https://docs.pydantic.dev/latest/concepts/models/#fields-with-non-hashable-default-values

I'll change.